.github/workflows/ci.yml

```diff
@@ -104,8 +104,12 @@ jobs:
 
   make-httpd-maintainer-mode:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        image-tag: [ "latest", "oldest-supported" ]
```

Comment:
BTW I changed my opinion on using arrays and I prefer itemizations now as these can be commented out per member or changed without changing the whole line containing all options. But decide as you prefer, e.g. for things that do never change, it doesn't matter.

Reply:
We have it in a few places. I don't really care, but if you do, I would change everywhere.